core/server/api_container/server/startosis_engine/startosis_executor.go

```diff
@@ -75,15 +75,17 @@ func (executor *StartosisExecutor) Execute(ctx context.Context, dryRun bool, par
 			}
 		}
 
-		scriptWithValuesReplaced, err := magic_string_helper.ReplaceRuntimeValueInString(serializedScriptOutput, executor.runtimeValueStore)
-		if err != nil {
-			propagatedErr := stacktrace.Propagate(err, "An error occurred while replacing the runtime values in the output of the script")
-			serializedError := binding_constructors.NewStarlarkExecutionError(propagatedErr.Error())
-			starlarkRunResponseLineStream <- binding_constructors.NewStarlarkRunResponseLineFromExecutionError(serializedError)
-			starlarkRunResponseLineStream <- binding_constructors.NewStarlarkRunResponseLineFromRunFailureEvent()
-			return
+		if !dryRun {
```

Comment:
can we add a --dry-run test in the CI for something that has runtime values?

Reply:
true; let me do that in some other PR. I will probably just use some existing package and add that in the CI.